docs/clusterfuzzlite/build_integration.md

```diff
@@ -14,13 +14,20 @@ permalink: /clusterfuzzlite/build-integration/
 ---
 
 ## Prerequisites
-ClusterFuzzLite supports statically linked [libFuzzer targets]({{ site.baseurl }}/reference/glossary/#fuzz-target) on Linux.
+ClusterFuzzLite supports statically linked
+[libFuzzer targets]({{ site.baseurl }}/reference/glossary/#fuzz-target) on
+Linux.
+
+We re-use the [OSS-Fuzz](https://github.com/google/oss-fuzz) toolchain to make
+building easier. If you are familiar with this, most of the concepts here are
+exactly the same, with one key difference. Rather than checking out the source
+code in the [`Dockerfile`](#dockerfile) using `git clone`, the `Dockerfile`
+copies in the source code directly during `docker build`.
```

Comment:
Is there anything else missing?

Reply:
I don't think so.
There's still the project.yaml difference which I haven't decided on yet. I think this is OK for now.